tests/Microsoft.DotNet.Framework.Docker.Tests/DockerHelper.cs

```diff
@@ -191,6 +191,10 @@ private string ExecuteWithLogging(string args, bool ignoreErrors = false, bool a
             ExecuteWithLogging(
                 $"inspect -f \"{{{{(index (index .NetworkSettings.Ports \\\"{containerPort}/tcp\\\") 0).HostPort}}}}\" {container}");
 
+        public string GetImageShell(string imageTag) =>
+            ExecuteWithLogging(
+                "inspect -f \"{{ .Config.Shell }}\" " + imageTag);
```

Comment:
Consider utilizing string interpolation.
```suggestion
                $"inspect -f \"{{{{ .Config.Shell }}}}\" {imageTag}");
```